### ISSUE TITLE
Could not resolve message field imported

### ISSUE THREAD

**Issue by zhiqingchen** [mafintosh/protocol-buffers#37]
decode throw error
not support import or do something wrong?

**Comment by geon** [comment]
I have this issue too. Just concatenating the files worked. :-/

**Comment by mingfai** [comment]
+1

**Comment by mog422** [comment]
I submitted PR #45

**Comment by luciopaiva** [comment]
Is mafintosh still working on this project? Some articles on the internet make reference to this library, but it seems abandoned. There are a handful of pull requests not accepted for a few months now :-(

**Comment by mafintosh** [comment]
@luciopaiva yes still very much maintaining this (latest commit is only a month old). commented on the pr.
